<launch>
    <node pkg="local" type="local" name="local" output="screen">
      <param name="localization_debug" type="bool" value = "true"/>
      <param name="use_steer" type="bool" value = "false"/>
      <param name="gps_err_fix" type="double" value = "0.005"/>
      <param name="gps_yaw_err_fix" type="double" value = "0.008"/>
      <param name="slam_err_fix" type="double" value = "0.005"/>
      <param name="slam_err_normal" type="double" value = "0.005"/>
      <param name="slam_yaw_err_fix" type="double" value = "0.005"/>
      <param name="slam_yaw_err_normal" type="double" value = "0.005"/>
      <param name="slam_fix_param" type="double" value = "0.15"/>
      <param name="slam_error_param" type="double" value = "0.1"/>
      <param name="speed_err" type="double" value = "0.001"/>
      <param name="cali_x_" type="double" value = "0.46"/>
      <param name="cali_y_" type="double" value = "-0.32"/>
    </node>
</launch>
